src/components/znkSession/components/sessionBtn/znkSession.component.js

```diff
@@ -0,0 +1,38 @@
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra.znkSession')
+        .component('znkSession', {
+            bindings: {},
+            templateUrl: 'components/znkSession/components/sessionBtn/znkSession.template.html',
+            controllerAs: 'vm',
+            controller: function ($scope, $log, $mdDialog, SessionSrv) {
+                'ngInject';
+
+                var vm = this;
+
+                this.$onInit = function() {
+                    vm.activeSessionGuid = {};
+                    vm.isLiveSessionActive = false;
+                    vm.endSession = SessionSrv.endSession;
+
+                    SessionSrv.getLiveSessionGUID().then(function (currSessionGuid) {
+                        vm.activeSessionGuid = currSessionGuid;
+                    });
+
+                    $scope.$watch('vm.activeSessionGuid', function (newLiveSessionGUID) {
+                        vm.isLiveSessionActive = newLiveSessionGUID.guid ? true : false;
```

Comment:
actually maybe you don't need this watch....

you can bind the ui to newLiveSessionGUID.guid